src/browser/web-view.js

```diff
@@ -221,18 +221,6 @@ const SelectAnimationAction = action =>
     }
   );
 
-const UnselectAnimationAction = action =>
-  ( action.type === "End"
-  ? Unselected
```

Comment:
I just noticed we no longer trigger `Unselected` action which parent component may be expecting (can't remember right now).